walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -11,13 +11,119 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
 package com.google.sps;
-
+ 
+import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.Collection;
-
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+ 
 public final class FindMeetingQuery {
+
+  private List<TimeRange> fullDay = new ArrayList<TimeRange>();
+  
+  // cannot modify arraylist in an enhanced for loop, must create separate arraylists
+  private List<TimeRange> freeTimes = new ArrayList<TimeRange>();
```

Comment:
Making these variables members of the class seems strange. Also you seem to be clearing them after each use. It would be better to declare them inside the each method where they are used, so you wouldn't need to worry about clearing them.

Reply:
I restructured my code with helper functions that rely on these to be class variables, please have a look and let me know if I should just pass them as params instead.

Reply:
It would be better to pass these in as parameters than to have them as class variables.